webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/RuleApi.scala

```diff
@@ -126,18 +126,19 @@ class RuleApi(
     val schema = API.CreateRule
     val restExtractor = restExtractorService
     def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
-      if(req.json_?) {
-        req.json match {
-          case Full(arg) =>
-            val restRule = restExtractor.extractRuleFromJSON(arg)
-            apiV2.createRule(restRule,req)
-          case eb:EmptyBox=>
-            toJsonError(None, JString("No Json data sent"))("createRule",restExtractor.extractPrettify(req.params))
-        }
-      } else {
-        val restRule = restExtractor.extractRule(req.params)
-        apiV2.createRule(restRule, req)
+      implicit var action = "createRule"
```

Comment:
no `implicit var` please, each one is terrible for understanding, both together... 
Please only `implicit val`